### PR TITLE
Update main sample page with new CKEditor logo

### LINKED ISSUE #1337: Update samples with a new logo
## Are you reporting a feature request or a bug?

Task

## Provide detailed reproduction steps (if any)

It could be nice to update our samples with new logo. Main colour of page could be also changed to integrate better with new logo.

### PR DESCRIPTION
**Pull request by msamsel**
## What is the purpose of this pull request?

task

## Does your PR contain necessary tests?

All patches which change the editor code must include tests. You can always read more
on [PR testing](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_contributing_code-section-tests),
[how to set the testing environment](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_tests) and
[how to create tests](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_tests-section-creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [ ] Manual tests

## What changes did you make?

- replace logo in sample to be new one
- update css colours in sample.

close #1337

### CODE REVIEW

**Comment by msamsel** [comment]
I had to correct path to `lesshat`, because `grunt less` was not working. After path update in less config, css might be easily update from less files with `grunt less` command.

File: samples/css/samples.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
```
Comment (mlewand) [inline samples/css/samples.css]: Please revert this change, it's not a subject of this issue. It's done in other PR, and your change here will cause merging conflicts.

Remove it by editing your commit history, as if the change never happened.

File: samples/css/samples.css [hunk 2]
```diff
@@ -1177,6 +1177,18 @@ header .balloon-a {
     display: none;
   }
 }
+.header-a .header-a-logo img {
+  width: 160px;
+  height: 60px;
+}
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
```
Comment (mlewand) [inline samples/css/samples.css]: Why did these headers leaked here? 😕

File: samples/less/samples.less [hunk 3]
```diff
@@ -2,7 +2,7 @@
 @components-dir: "../../node_modules/cksource-samples-framework/components";
 
 // Good 'ol Lesshat.
-@import "../../node_modules/cksource-samples-framework/node_modules/lesshat/build/lesshat";
+@import "../../node_modules/lesshat/build/lesshat";
```
Comment (mlewand) [inline samples/less/samples.less]: If we were to use it that way we need to add `lesshat` as a dev dependency to our project too.

File: samples/css/samples.css (hunk 2 above)
Comment (msamsel) [reply samples/css/samples.css]: I haven't touch manually `.css` file, so all changes was made by `grunt less` script.

File: samples/less/samples.less (hunk 3 above)
Comment (msamsel) [reply samples/less/samples.less]: I'm pretty sure that I get error multiple times because this file wasn't present. It seems that I had some repository problem after reinstallation it's there.

**Comment by msamsel** [comment]
SVG added with fallback for not supported browsers. Replace image to be exact size, as now we have svg for hidpi. Update styles to scale svg to proper size. Restore header update in `samples.css` which was generated automatically by `grunt less`.

File: samples/less/samples.less (hunk 3 above)
Comment (mlewand) [reply samples/less/samples.less]: It depends on npm resolution. Currently node will put deps of your deps in your `node_modules` directory, but will that be the case in a year from here? We don't know.

Thus it needs to be added to deps.

File: CHANGES.md [hunk 4]
```diff
@@ -25,10 +25,15 @@ Fixed Issues:
...
 * [#1010](https://github.com/ckeditor/ckeditor-dev/issues/1010): Fixed: CSS `border` shorthand property was incorrectly expanded ignoring `border-color` style.
 
+
 API Changes:
 
 * [#1346](https://github.com/ckeditor/ckeditor-dev/issues/1346): [Balloon Toolbar](https://ckeditor.com/cke4/addon/balloontoolbar) [context manager API](https://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR.plugins.balloontoolbar.contextManager) is now available in [pluginDefinition.init](https://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR.pluginDefinition-method-init) method of a [requiring](https://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR.pluginDefinition-property-requires) plugin.
 
+Other Changes:
+
+* [#1337](https://github.com/ckeditor/ckeditor-dev/issues/1337): Update `samples` layout with actuall CKEditor logo and coherent color scheme.
```
Comment (mlewand) [inline CHANGES.md]: actuall -> actual

File: samples/less/samples.less (hunk 3 above)
Comment (msamsel) [reply samples/less/samples.less]: I wrongly interpret the previous comment then ;)

**Comment by msamsel** [comment]
* add devDepndancy
* correct path to lesshat in our less file
* regenerate styles.css to have sure that script works properly
* package.json was also sorted when `npm i -D ...` was used

**Comment by msamsel** [comment]
I completely forget about toolbar configuration :(
